js/app.js

```diff
@@ -5,95 +5,115 @@ let username = prompt('What is your name?');
 
 alert("Hello " + username + "!");
 
-let question1 = prompt('Was my first job after graduation an accounting clerk?').toLowerCase();
-
-if (question1 === 'yes' || question1 === 'y') {
-  alert('Yes is correct!');
-  score++;
-} else {
-  alert('Sorry, no is not the correct response.');
-}
-
-let question2 = prompt('Have I ever worked part time at the Seattle Sonics Team Shop at Key Arena?').toLowerCase();
-
-if (question2 === 'yes' || question2 === 'y') {
-  alert('You\'re right!');
-  score++;
-} else {
-  alert('Sorry, that is incorrect');
+function question1() {
+  let question1 = prompt('Was my first job after graduation an accounting clerk?').toLowerCase();
+
+  if (question1 === 'yes' || question1 === 'y') {
+    alert('Yes is correct!');
+    score++;
+  } else {
+    alert('Sorry, no is not the correct response.');
+  }
 }
+question1();
```

Comment:
Try to call your functions all in one place. The standard spot to do so is at the end of your document. Having them all in one spot makes your code more readable, and more easily debugged.